### ISSUE TITLE
Option to generate HDR latency distribution files

### ISSUE THREAD

**Issue by pveentjer** [scylladb/scylla-cluster-tests#3133]
I would like the have the ability to generate and download the HDR latency distribution file. There is a special option on cassandra-stress:

```-log hdrfile=somefile.hdr```

This will generate the latency distribution file and it should be downloaded.

The reason this file is needed it that it is the only way to correctly determine the latency percentiles if there are multiple load generators. Currently the average of these values is calculated, but this is incorrect and it will lead to too low percentiles.

For more information see: http://pveentjer.blogspot.com/2017/08/percentiles-and-mean.html

HDR Histogram has a special option to merge latency distribution files and then you can correctly determine the percentiles.

**Comment by dkropachev** [comment]
Hello Peter, 

Here is branch that has this feature: 
git@github.com:dkropachev/scylla-cluster-tests.git:cs_hdrfile

Way it works:
1. Make sure that cs_dump_hdr is set to true. Currently it is turned on for any run
2. Run the test
3. If it is local - go to the loader log folder and find there *.hdr files
4. It if is jenkins - get loader log zip file and find *.hdr files there

Unfortunately, by some reason cs produce only 292 bytes header, i'am trying to find a reason and fix it.
Example: https://cloudius-jenkins-test.s3.amazonaws.com/2503dc5f-2041-4845-b2c0-6b28fac0f82b/20210115_133916/loader-set-2503dc5f.zip

Let me know if you need help on running it via jenkins.

**Comment by pveentjer** [comment]
Are you planning to make an PR so it ends up in master?

**Comment by dkropachev** [comment]
> Are you planning to make an PR so it ends up in master?

Yes, but we need to get hdr data produces and remove cs_dump_hdr: true from defaults and set it for runs you are interested in.

**Comment by soyacz** [comment]
it's already done
